scripts/performance_testing/README.md

```diff
@@ -0,0 +1,211 @@
+# Performance Testing 
```

Comment:
quick q: we dont want to put this doc in `docs`?

Reply:
It's a little bit internal so I was steering away from the docs directory?